### ISSUE TITLE
Pip install of neutcurve doesn't work on python 3.8

### ISSUE THREAD

**Issue by AoifeHughes** [jbloomlab/neutcurve#26]
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-11-82cfebc07ded> in <module>
      1 import pandas as pd
      2 import numpy as np
----> 3 import neutcurve
      4 from neutcurve.colorschemes import CBMARKERS, CBPALETTE

~/opt/anaconda3/envs/playground/lib/python3.8/site-packages/neutcurve/__init__.py in <module>
     20 __url__ = 'https://github.com/jbloomlab/neutcurve'
     21 
---> 22 from neutcurve.curvefits import CurveFits  # noqa: F401
     23 from neutcurve.hillcurve import HillCurve  # noqa: F401

~/opt/anaconda3/envs/playground/lib/python3.8/site-packages/neutcurve/curvefits.py in <module>
     10 import math
     11 
---> 12 import dmslogo.facet
     13 import dmslogo.utils
     14 

~/opt/anaconda3/envs/playground/lib/python3.8/site-packages/dmslogo/__init__.py in <module>
     21 from dmslogo.facet import facet_plot  # noqa: F401
     22 from dmslogo.line import draw_line  # noqa: F401
---> 23 from dmslogo.logo import draw_logo  # noqa: F401

~/opt/anaconda3/envs/playground/lib/python3.8/site-packages/dmslogo/logo.py in <module>
     41 for _fontfile in matplotlib.font_manager.findSystemFonts(None):
     42     try:
---> 43         matplotlib.font_manager.fontManager.addfont(_fontfile)
     44     except RuntimeError:
     45         # problem with loading emoji fonts; solution here is just to

~/opt/anaconda3/envs/playground/lib/python3.8/site-packages/matplotlib/font_manager.py in addfont(self, path)
   1098         else:
   1099             font = ft2font.FT2Font(path)
-> 1100             prop = ttfFontProperty(font)
   1101             self.ttflist.append(prop)
   1102 

~/opt/anaconda3/envs/playground/lib/python3.8/site-packages/matplotlib/font_manager.py in ttfFontProperty(font)
    462         return 500  # "medium", not "regular"!
    463 
--> 464     weight = int(get_weight())
    465 
    466     #  Stretch can be absolute and relative

~/opt/anaconda3/envs/playground/lib/python3.8/site-packages/matplotlib/font_manager.py in get_weight()
    445         try:
    446             ps_font_info_weight = (
--> 447                 font.get_ps_font_info()["weight"].replace(" ", "") or "")
    448         except ValueError:
    449             pass

TypeError: tuple indices must be integers or slices, not str

```

**Comment by jbloom** [comment]
I just tried this, and didn't get the error. I did:
```
conda create --name test_neutcurve python=3.8 pip
conda activate test_neutcurve
pip install neutcurve
```
and then I could import `neutcurve` just fine.

My `conda` environment file is attached.

Can you create a `conda` environment that gives your error to help me troubleshoot? (Is it possible that the problem is you have an old version of `matplotlib`?).

[environment.yml.txt](https://github.com/jbloomlab/neutcurve/files/6117779/environment.yml.txt)

**Comment by AoifeHughes** [comment]
Hi, 

Nope, everything is up-to-date. 

```
(test_neutcurve) nathanhughes@Nathans-MacBook-Pro ~ % conda env export
name: test_neutcurve
channels:
  - defaults
dependencies:
  - ca-certificates=2021.1.19=hecd8cb5_1
  - certifi=2020.12.5=py38hecd8cb5_0
  - libcxx=10.0.0=1
  - libedit=3.1.20191231=h1de35cc_1
  - libffi=3.3=hb1e8313_2
  - ncurses=6.2=h0a44026_1
  - openssl=1.1.1j=h9ed2024_0
  - pip=21.0.1=py38hecd8cb5_0
  - python=3.8.8=h88f2d9e_4
  - readline=8.1=h9ed2024_0
  - setuptools=52.0.0=py38hecd8cb5_0
  - sqlite=3.33.0=hffcf06c_0
  - tk=8.6.10=hb0a8c7a_0
  - wheel=0.36.2=pyhd3eb1b0_0
  - xz=5.2.5=h1de35cc_0
  - zlib=1.2.11=h1de35cc_3
  - pip:
    - cycler==0.10.0
    - dmslogo==0.6.2
    - kiwisolver==1.3.1
    - matplotlib==3.3.4
    - neutcurve==0.5.3
    - numpy==1.20.1
    - palettable==3.3.0
    - pandas==1.2.3
    - pillow==8.1.2
    - pyparsing==2.4.7
    - python-dateutil==2.8.1
    - pytz==2021.1
    - pyyaml==5.4.1
    - scipy==1.6.1
    - six==1.15.0
    - xlrd==2.0.1
prefix: /Users/nathanhughes/opt/anaconda3/envs/test_neutcurve
```

I copied exactly what you have written above in creating a new environment. I also tried 3 versions of matplotlib (<3, 3 and current)

**Comment by jbloom** [comment]
OK, then it must somehow relate to how the font files are installed on your computer. I apologize that this isn't something I can troubleshoot at this point. See here: https://github.com/jbloomlab/neutcurve/issues/25

**Comment by jbloom** [comment]
Should be fixed in #30, which is version 0.5.5.
